lib/sablon/html/ast.rb

```diff
@@ -8,6 +8,26 @@ def accept(visitor)
       def self.node_name
         @node_name ||= name.split('::').last
       end
+
+      private
+
+      # processes attributes defined on the node into wordML property syntax
+      def process_properties
```

Comment:
It would also be nice to have a specific test-case exercising the ParagraphProperties class.